scheduler/classes/Day.py

```diff
@@ -31,6 +31,8 @@ def from_str(day_name_str: str) -> DayName:
     if day_name_str == "SUNDAY":
         return DayName.SUNDAY
 
+    # TODO: defualt return
+
```

Comment:
Are these just to remind us to do it later?

Reply:
Just a reminder.